src/System.Data.Common/tests/System/Data/DataTableTest.cs

```diff
@@ -388,6 +389,25 @@ public void SelectOperators()
 
         }
 
+        [Fact]
+        public void DataColumnTypeSerialization()
+        {
+            DataTable dt = new DataTable("MyTable");
+            DataColumn dc = new DataColumn("dc", typeof(Int32));
+            dt.Columns.Add(dc);
+            dt.RemotingFormat = SerializationFormat.Binary;
+
+            DataTable dtDeserialized;
+            using (MemoryStream ms = new MemoryStream())
+            {
+                BinaryFormatter bf = new BinaryFormatter();
+                bf.Serialize(ms, dt);
+                ms.Seek(0, SeekOrigin.Begin);
+                dtDeserialized = (DataTable)bf.Deserialize(ms);
+            }
+            Assert.Equal(dc.DataType.ToString(), dtDeserialized.Columns[0].DataType.ToString());
```

Comment:
Why do you need to call .ToString() on both types? I expect Assert.Equal to be able to check for equality of two types.